index.js

```diff
@@ -1,6 +1,6 @@
 const fs = require('fs');
 const traversy = require('traversy');
-const mime = require('mime');
+const { lookup } = require('mrmime');
```

Comment:
Because the word `lookup` is so generic, perhaps we should not destruct it and handle it like it was before where it's `const mime = require('mrmime');` and then where we use lookup `mime.lookup(...)`

Reply:
Good idea. Updated